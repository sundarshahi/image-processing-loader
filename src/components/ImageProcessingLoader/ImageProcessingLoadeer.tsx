function ImageProcessingLoader() {
  return <div>Loading...</div>;
}

export default ImageProcessingLoader;
