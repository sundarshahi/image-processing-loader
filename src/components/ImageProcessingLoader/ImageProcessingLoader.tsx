export function ImageProcessingLoader() {
  return <div>Loading...</div>;
}
