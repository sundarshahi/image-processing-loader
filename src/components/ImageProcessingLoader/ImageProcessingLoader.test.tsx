import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import ImageProcessingLoader from "./ImageProcessingLoader";

describe("ImageProcessingLoader", () => {
  it("render ImageProcessingLoader correctly", async () => {
    const { container } = render(<ImageProcessingLoader />);
    expect(container).toMatchSnapshot();
  });
});
