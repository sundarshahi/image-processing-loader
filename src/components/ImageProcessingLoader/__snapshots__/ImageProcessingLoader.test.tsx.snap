// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ImageProcessingLoader > render ImageProcessingLoader correctly 1`] = `
<div>
  <div>
    Loading...
  </div>
</div>
`;
